CONTRIBUTING.md

```diff
@@ -0,0 +1,248 @@
+# Contributing to Lux
+
+Thank you for your interest in contributing to Lux! This document will help
+answer any outstanding questions you may have about the contribution process.
+
+*Please read the [Code of Conduct](./CODE_OF_CONDUCT.md) before you participate
+in community.*
+
+## Contents
+
```

Comment:
I have added a [Ways to Contribute](https://github.com/postlight/lux/blob/docs-contributing/CONTRIBUTING.md#ways-to-contribute) section. I had a tough time wording it for some reason. For the sake of tone, I tried to express that the list is not exhaustive without directly saying it. I'm not sure that came across correctly. Suggestions/feedback would be appreciated. 😃